Destiny.ScrimTracker.Api/Controllers/GuardianController.cs

```diff
@@ -5,55 +5,90 @@
 using System.Threading.Tasks;
 using Destiny.ScrimTracker.Api.Requests;
 using Destiny.ScrimTracker.Logic.Models;
+using Destiny.ScrimTracker.Logic.Repositories;
 using Destiny.ScrimTracker.Logic.Services;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 
 namespace Destiny.ScrimTracker.Api.Controllers
 {
     [ApiController]
     [Route("[controller]")]
-    public class GuardianController : ControllerBase
+    public class GuardianController : Controller
     {
+        private readonly IMatchService _matchService;
         private readonly IGuardianService _guardianService;
         private readonly ILogger<GuardianController> _logger;
 
-        public GuardianController(IGuardianService guardianService, ILogger<GuardianController> logger)
+        public GuardianController(IMatchService matchService, IGuardianService guardianService, ILogger<GuardianController> logger)
         {
+            _matchService = matchService;
             _guardianService = guardianService;
             _logger = logger;
         }
 
         [HttpGet]
-        public IEnumerable<Guardian> Get()
+        public IActionResult Get()
         {
-            return _guardianService.GetGuardians();
+            var guardianSnapshots = _guardianService.GetGuardians();
+            return View(guardianSnapshots);
         }
 
-        [HttpGet("{guardianId}")]
-        public Guardian GetGuardian(string guardianId)
+        [HttpGet("/add_guardian")]

```

Comment:
If we're using REST this should be `guardians/new`
```suggestion
        [HttpGet("/guardians/new")]
```

Reply:
ah. yea good call.